apps/next/lib/constants.ts

```diff
@@ -22,3 +22,5 @@ export function queueNameForEnv(env: string) {
 }
 
 export const hetznerRedHex = "#D50C2D";
+
+export const whiteishHex = "#f8f8f8";
```

Comment:
can you make this F8FAFC to match globals.css foreground color for dark theme, and then maybe comment along these lines to keep it in sync

Reply:
Done.